src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -862,10 +862,10 @@
       [attachment-filename component-id question-group-idx attachment-idx true]]
      [:div.application__form-attachment-list-item-sub-container.application__form-attachment-error-container
       (doall
-       (map-indexed (fn [i [error]]
+       (map-indexed (fn [i [error params]]
                       ^{:key (str "attachment-error-" i)}
                       [:span.application__form-attachment-error
-                       (tu/get-hakija-translation error lang)])
+                       (tu/get-hakija-translation error lang params)])
```

Comment:
Kaikilla virheillä ei näyttäis olevan parametrejä, jolloin `params` on `nil`. Otsan kirkkaudella en osaa sanoa toimiiko tuo `get-hakija-translation` silloin oikein.

Reply:
Mietin tätä korjatessa ihan samaa, ja testasin toimivaksi, paremmin sanoen en saanut muita virheviestejä hajoamaan. Tuo `tu/get-hakija-translation` on sisäiseltä logiikaltaan nil-suojattu. Esimerkkinä tästä kelvannee tämä screenshot samaisesta UI:sta, jossa on houkuteltu esiin myös staattisen tekstin sisältävät virheviestit, jonka olisin voinut laittaa toki jo alunperinkin tuohon ekaan kuvatukseen mukaan:

![Screenshot 2020-03-11 at 15 32 58](https://user-images.githubusercontent.com/1393900/76422312-abbcf180-63ad-11ea-875b-0568a6c555de.png)